peers/info_client.go

```diff
@@ -48,7 +48,8 @@ func (i *InfoAPI) GetNodeID(ctx context.Context) (ids.NodeID, *signer.ProofOfPos
 }
 
 func (i *InfoAPI) GetNodeIP(ctx context.Context) (string, error) {
-	return i.client.GetNodeIP(ctx, i.options...)
+	addrPort, err := i.client.GetNodeIP(ctx, i.options...)
+	return addrPort.Addr().String(), err
```

Comment:
Let's have `InfoAPI` continue to be a passthrough to `InfoClient`, and have `GetNodeIP` have the same return values as `InfoClient`'s method.

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/373/commits/641e5326b88389f0647a4d74781366343c8fb37a